src/domain/common/canvas/canvas.entity.ts

```diff
@@ -33,4 +34,12 @@ export class Canvas extends NameableEntity implements ICanvas {
   })
   @JoinColumn()
   checkout?: CanvasCheckout;
+
+  @OneToOne(() => Visual, {
+    eager: true,
```

Comment:
don't do eager

Reply:
In a pending PR the TemplateInfo is no longer eager loaded.

Reply:
A single query can be run from the client to get the URIs for all canvas templates. Does that answer your question?

Reply:
No, eager tells the orm to load this relation everytime the canvas has been loaded. It's not graphql related.

Reply:
No, and yes.
No - the change you made does not produce this result.
Yes - naive graphql implementation does this and it's addressed with data loaders.